network/endpoint_windows.go

```diff
@@ -5,12 +5,25 @@ package network
 
 import (
 	"encoding/json"
+	"fmt"
 	"net"
 	"strings"
 
 	"github.com/Azure/azure-container-networking/log"
 	"github.com/Azure/azure-container-networking/network/policy"
 	"github.com/Microsoft/hcsshim"
+	"github.com/Microsoft/hcsshim/hcn"
+)
+
+const (
+	// hcnSchemaVersionMajor indicates major version number for hcn schema
+	hcnSchemaVersionMajor = 2
+
+	// hcnSchemaVersionMinor indicates minor version number for hcn schema
+	hcnSchemaVersionMinor = 0
+
+	// hcnIpamTypeStatic indicates the static type of ipam
+	hcnIpamTypeStatic = "Static"
 )
 
 // HotAttachEndpoint is a wrapper of hcsshim's HotAttachEndpoint.
```

Comment:
check err before useHnsV2, also this looks common, maybe we can turn this into a separate func